application/comit_node/src/bin/comit_node.rs

```diff
@@ -173,10 +174,12 @@ fn spawn_comit_i_instance(settings: &ComitNodeSettings, runtime: &mut tokio::run
 }
 
 fn auth_origin(settings: &ComitNodeSettings) -> String {
-    match &settings.web_gui {
+    let auth_origin = match &settings.web_gui {
         Some(http_socket) => format!("http://localhost:{}", http_socket.port),
-        None => "http://localhost:8080".to_string(),
-    }
+        None => "http://localhost:3000".to_string(),
+    };
+    log::info!("Auth origin enabled on: {}", auth_origin);
```

Comment:
I think we can either undo this or change it to `debug` as this was just for testing 😬